.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
- Nicely done on adding these 2 files to the .gitignore file, keep the great work 👍

Reply:
Thank you